src/app/item-page/simple/field-components/specific-field/item-page-field.component.ts

```diff
@@ -51,6 +51,8 @@ export class ItemPageFieldComponent {
      */
     urlRegex?: string;
 
+  img?: string;
```

Comment:
Again, please add a comment above this to describe the input.  (Also this appears to be misaligned compared to surrounding code)